LocalSettings.php.template

```diff
@@ -49,8 +49,7 @@ ${DOLLAR}wgRCMaxAge = 365 * 24 * 3600;
 wfLoadSkin( 'Vector' );
 
 ## Wikibase
-# Load Wikibase repo & client with the example / default settings.
-require_once "${DOLLAR}IP/extensions/Wikibase/repo/Wikibase.php";
+wfLoadExtension( 'WikibaseRepository', "$IP/extensions/Wikibase/extension-repo.json" );
 require_once "${DOLLAR}IP/extensions/Wikibase/repo/ExampleSettings.php";
-require_once "${DOLLAR}IP/extensions/Wikibase/client/WikibaseClient.php";
-require_once "${DOLLAR}IP/extensions/Wikibase/client/ExampleSettings.php";
+wfLoadExtension( 'WikibaseClient', "$IP/extensions/Wikibase/extension-client.json" );
```

Comment:
Maybe also with ${DOLLAR}?

Reply:
standard initialization from [Wikibase](https://www.mediawiki.org/wiki/Wikibase/Installation) is without dollar and it is working like this, so this is sufficient atm.

Reply:
If there is an env var called `$IP`, envsubst in `entrypoint.sh` will substitute it. It's probably not the case, but to protect `$IP`, I'd still use `${DOLLAR}IP`. See https://stackoverflow.com/questions/24963705/is-there-an-escape-character-for-envsubst/24964089#24964089

Cf. here
https://github.com/MaRDI4NFDI/docker-wikibase/blob/bba24aefa1f9d4638bf84da86f4ebef782eb7102/entrypoint.sh#L40-L41